src/pages/assets.tsx

```diff
@@ -93,245 +32,253 @@ const AssetsPage = ({ data }: PageProps<Queries.AssetsPageQuery, Context>) => {
     ? data.ethDiamondPurpleHero
     : data.ethDiamondBlackHero
   return (
-    <Page>
+    <Flex direction={"column"} width="full">
```

Comment:
To follow conventions, lets not wrap the value if it is a string literal (this applies also to the rest of the changes).
```suggestion
    <Flex direction="column" width="full">
```